cernopendata/mappings/v7/records/record-v1.0.0.json

```diff
@@ -51,12 +51,7 @@
               }
             },
             "orcid": {
-              "type": "text",
-              "fields": {
-                "exact": {
-                  "type": "keyword"
-                }
-              }
+              "type": "keyword"
```

Comment:
I get 10 results when I try from scratch, so it's even better than your screenshot :wink: